frontend/src/calendar/displayDate.tsx

```diff
@@ -64,7 +64,7 @@ export function toDojoDateString(
 export function toDojoTimeString(
     date: Date,
     timezone: string | undefined,
-    timeFormat: TimeFormat | undefined,
+    timeFormat?: TimeFormat,
```

Comment:
Can you revert this change as well? `timeFormat: TimeFormat | undefined` has different semantics than `timeFormat?: TimeFormat`. I want to ensure that if you call this function, you are passing in a time format, which may possibly be undefined. The latter allows you to not pass in a time format at all

Reply:
My mistake for missing it in the revert. Allowing not passing in whatever it was for the other change I reverted was indeed the intent, for it's usage in the link cards.